### PR TITLE
Set interval branch

### PR DESCRIPTION
**Pull request by adrienzag**
added functions/class to allow for setting of update interval

### CODE REVIEW

**Comment by adrienzag** [comment]
- created SetUpdateInterval class that will reset/set the interval for our update function

- under the WeatherByLocation.java (line 67) our above associated command is mapped to "set-weather-update-interval", the executor is mapped to our class SetUpdateInterval.java

- under the plugins.yml we have added set-weather-update-interval and its description,usage and permission so that when we make calls through the terminal, the plugin will know where to look

- "minutes-between-updates" in the config.yml is the field to be edited

File: src/main/java/mc/spigot/weatherbylocation/ServerLocator.java
Comment (scottbarnesg) [inline src/main/java/mc/spigot/weatherbylocation/ServerLocator.java:1]: I did some refactoring of the ServerLocator class - please back out these changes, as they are unrelated to this issue.

**Comment by scottbarnesg** [comment]
@adrien2-public thanks for submitting this MR! Overall it looks good, I will pull and test on my test server.

I did some refactoring of the WeatherByLocation and ServerLocator class, and resolved the conflicts between your branch and the master branch. Looks like there may be some outstanding changes in the ServerLocator class that needs to be removed.

To help prevent these conflicts in future PRs, I recommend always creating new feature branches from the master branch. My workflow is:

```
git checkout master
git pull
git checkout -b new-branch-name
```

File: src/main/java/mc/spigot/weatherbylocation/ServerLocator.java
Comment (adrienzag) [reply src/main/java/mc/spigot/weatherbylocation/ServerLocator.java:1]: I see, i have synced your changes to my master branch and pushed/merged those changes downstream to my other branches.

This should be reflected in the SetIntervalBranch branch.

**Review by scottbarnesg — APPROVED** [review]
Pulled and confirmed this works as expected on my test server. Looks good to me!

**Pull request merged**